src/clu_log.c

```diff
@@ -50,11 +50,8 @@ void wolfCLU_OutputON(void)
 
 void DefaultLoggingCb(int logLevel, const char *const msgStr)
 {
-    if (loggingEnabled && loggingLevel <= logLevel) {
-        printf("%s\r\n", msgStr);
-    }
-
-    if (logLevel == WOLFCLU_E0) {
+    if ((loggingEnabled && loggingLevel <= logLevel) ||
```

Comment:
this puts info and debug level logs to stderr too .... is that the preferred logging for the client?

Reply:
My understanding is that logging/diagnostic messages [should go to stderr, not stdout](https://stackoverflow.com/questions/4919093/should-i-log-messages-to-stderr-or-stdout). The wolfSSL default logging callback goes to stderr, for what it's worth.